plugins/win-capture/game-capture.c

```diff
@@ -1867,40 +2069,70 @@ static const char *game_capture_name(void *unused)
 
 static void game_capture_defaults(obs_data_t *settings)
 {
-	obs_data_set_default_string(settings, SETTING_MODE, SETTING_MODE_ANY);
+	obs_data_set_default_string(settings, SETTING_MODE, SETTING_MODE_AUTO);
 	obs_data_set_default_int(settings, SETTING_WINDOW_PRIORITY,
 				 (int)WINDOW_PRIORITY_EXE);
 	obs_data_set_default_bool(settings, SETTING_COMPATIBILITY, false);
 	obs_data_set_default_bool(settings, SETTING_FORCE_SCALING, false);
 	obs_data_set_default_bool(settings, SETTING_CURSOR, true);
 	obs_data_set_default_bool(settings, SETTING_TRANSPARENCY, false);
 	obs_data_set_default_string(settings, SETTING_SCALE_RES, "0x0");
-	obs_data_set_default_bool(settings, SETTING_LIMIT_FRAMERATE, false);
+	obs_data_set_default_bool(settings, SETTING_LIMIT_FRAMERATE, true);
```

Comment:
Why this value has been changed to true?

Reply:
Unlimited frame rate can have negative side effect when game frame rate is high. I think that for users who have Auto mode in game capture better to use this option enabled.

Reply:
Got it, I think this is a good idea.